wgpu-types/src/lib.rs

```diff
@@ -361,30 +434,49 @@ pub fn read_spirv<R: io::Read + io::Seek>(mut x: R) -> io::Result<Vec<u32>> {
 }
 
 bitflags::bitflags! {
+    /// Describes the shader stages that this binding will be visible from.
```

Comment:
"this binding" - but this bitflag exits independent of any binding. I'd either rename the struct `BindingShaderStage` to rework the comments to not mention bindings specifically, only shader stages